src/runtime/hexagon_remote/Makefile

```diff
@@ -57,7 +57,7 @@ CCFLAGS-arm-32-android = --sysroot ${ANDROID_NDK_ROOT}/platforms/android-21/arch
 CCFLAGS-host := ${CCFLAGS} -I ${HEXAGON_TOOLS_ROOT}/include/iss/ -fPIC \
 	-L${HEXAGON_TOOLS_ROOT}/lib/iss/ -lwrapper
 
-CCFLAGS-v60 := $(CCFLAGS-v60) ${COMMON_CCFLAGS} -I ${HEXAGON_SDK_INCLUDES}
+CCFLAGS-v60 := $(CCFLAGS-v60) ${COMMON_CCFLAGS} -I ${HEXAGON_SDK_INCLUDES} -I ${HEXAGON_SDK_ROOT}lib/common/qurt/ADSPv60MP/include
```

Comment:
This should be ${HEXAGON_SDK_LIBS}/common/..., in newer SDKs, the folder is "libs", not "lib". We hide this difference in that variable.